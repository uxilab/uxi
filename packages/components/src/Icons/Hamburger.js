import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Hamburger = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 16 16"
      width="24px"
      height="24px"
    >
      <g id="Layer_2" data-name="Layer 2">
        <g id="Layer_1-2" data-name="Layer 1">
          <rect width="16" height="1.73" />
          <rect y="7.14" width="16" height="1.73" />
          <rect y="14.28" width="16" height="1.72" />
        </g>
      </g>
    </svg>
  </SvgIcon>
);

Hamburger.propTypes = {
  color: PropTypes.string,
};

Hamburger.defaultProps = {
  color: '#6d6d71',
};

export default Hamburger;
