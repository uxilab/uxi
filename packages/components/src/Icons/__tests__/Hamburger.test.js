import React from 'react';
import Hamburger from '../Hamburger';

describe('Icon : <Hamburger />', () => {
  it('match snapshot', () => {
    expect(shallow(<Hamburger />)).toMatchSnapshot();
  });
});

