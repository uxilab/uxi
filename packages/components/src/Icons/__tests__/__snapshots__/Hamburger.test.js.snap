// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Hamburger /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 16 16"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <rect
          height="1.73"
          width="16"
        />
        <rect
          height="1.73"
          width="16"
          y="7.14"
        />
        <rect
          height="1.72"
          width="16"
          y="14.28"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
